hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestAsyncTableTracing.java

```diff
@@ -100,7 +99,9 @@
 
   private ClientService.Interface stub;
 
-  private AsyncConnection conn;
+  private User user;
```

Comment:
Do we really need to declare it as a class member field? It is only used in one method?

Reply:
Nope, I suppose not.